packages/mui-material/src/Stack/Stack.test.js

```diff
@@ -256,5 +256,67 @@ describe('<Stack />', () => {
         flexDirection: 'column',
       });
     });
+
+    it('should generate correct styles if custom breakpoints are provided in theme', () => {
+      expect(
+        style({
+          ownerState: {
+            direction: 'column',
+            spacing: 4,
+          },
+          theme: {
+            ...defaultTheme,
+            breakpoints: {
+              ...defaultTheme.breakpoints,
+              values: {
+                small: 375,
+              },
+            },
+          },
```

Comment:
`small` is a _new_ breakpoint. I added `small` breakpoint because this issue is occuring for that particular keyword as said in the description. `String` object has `small` method in it's prototype. There are many other methods of string but `small` exposed the issue as any developer could have a custom value with `small` keyword in it's breakpoint (like the issue reporter had it).

But anyway now I added new breakpoints more closely to how developers would do it and removed the default theme breakpoints spreading.